ppx_version.opam

```diff
@@ -1,8 +1,15 @@
 opam-version: "2.0"
+name: "ppx_version"
+authors: "opensource@o1labs.org"
+homepage: "https://github.com/o1-labs/ppx_version"
+bug-reports: "opensource@o1labs.org"
 version: "0.1"
-synopsis: ""
-description:
-  "OCaml extension point (ppx) meant to assure the stability of types and their Bin_prot serializations."
+descr: """
+Ocaml extension point (ppx) meant to assure the stability of types and their Bin_prot serializations
+
+Read the README for more information.
```

Comment:
It might be better to give a link to the readme, e.g. `See the readme at https://github.com/o1-labs/ppx_version/README.md for more information`

Reply:
done!